pkg/backends/newrelic/newrelic_test.go

```diff
@@ -402,6 +402,21 @@ func TestEventFormatter(t *testing.T) {
 	}
 }
 
+func TestNewRelicFulfillRunnerInterface(t *testing.T) {
```

Comment:
An alternative way of managing this is sticking a line like the following:

```
var _ = gostatsd.Runnable((*Client)(nil))
```

I typically put this directly under the `struct`, or at the very end of the file.

This forces a compile time check.  It's also a fairly common idiom, so the intent is clearer without needing to write a test, although we don't use it in this project.

Reply:
Thanks for the review! Yeah, I like to rely on the compiler for that checks.

I've just updated the PR, added the check and removed the test.